shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Shelley.hs

```diff
@@ -93,6 +93,8 @@ type instance Core.PParams (ShelleyEra c) = SPP.PParams (ShelleyEra c)
 
 type instance Core.Witnesses (ShelleyEra c) = WitnessSet (ShelleyEra c)
 
+type instance Core.PParamsDelta (ShelleyEra c) = SPP.PParamsUpdate (ShelleyEra c)
```

Comment:
This is already defined on line 75 above. The fact that it works being defined there is a ltitle scary, since the type family is no longer associated with the class, but if adding it in another place we should remove it from the line above.

Reply:
Oh, even worse! There are *two* `PParamsDelta`, since https://github.com/input-output-hk/cardano-ledger-specs/commit/c497f64ed73fd4c8783aa63835065c63e7634568 didn't delete the old one when the new one was added! We should definitely clean this up, otherwise it's very confusing (and potentially harmful)

Reply:
cleaned up.